website/addons/osfstorage/model.py

```diff
@@ -28,17 +30,15 @@ class OsfStorageNodeSettings(StorageAddonBase, AddonNodeSettingsBase):
     complete = True
     has_auth = True
 
-    root_node = fields.ForeignField('OsfStorageFileNode')
-    file_tree = fields.ForeignField('OsfStorageFileTree')
-
-    # Temporary field to mark that a record has been migrated by the
-    # migrate_from_oldels scripts
-    _migrated_from_old_models = fields.BooleanField(default=False)
+    root_node = fields.ForeignField('StoredFileNode')
 
     @property
     def folder_name(self):
         return self.root_node.name
 
+    def get_root(self):
```

Comment:
I think I understand what's going on and why, but this seems fragile. If a developer calls `<OsfStorageNodeSettings instance>.root_node`, they get a different object than if they call `<OsfStorageNodeSettings instance>.get_root()`.

I'm OK with it right now, I just wanted to poke at it.